### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by SoliatOmolara** [ModECI/MDF#335] [General]
Hello everyone, I'm a data scientist, machine learning engineer, and also an outreachy applicant. I'm new here @pgleeson I will like to know how to start contributing because it's a bit confusing for me.

**Comment by pgleeson** [comment]
Hi @LeehahData! Please have a look at https://github.com/ModECI/MDF/discussions/346 for an introduction to the project and seom suggested tasks.
